docker-compose.yml

```diff
@@ -15,3 +13,5 @@ services:
       # Note that this mounts the *entire* repo directory (including
       # files ignored in .dockerignore when building the image)
       - ./:/opt/app
+    tty: true
+    stdin_open: true
```

Comment:
are these changes necessary?

Reply:
these may not be necessary, but they do help with debugging.